packages/front-end/components/Experiment/BreakDownResults.tsx

```diff
@@ -68,6 +73,15 @@ const BreakDownResults: FC<{
       .map((metricId) => {
         const metric = getMetricById(metricId);
         const { newMetric } = applyMetricOverrides(metric, metricOverrides);
```

Comment:
So I went to check this after seeing the custom logic in SqlIntegration.ts, because it feels like you could use applyMetricOverrides and the Experiment interface to capture these two quantities of interest.  Am I missing something?